src/archivist/recorder/repositories/git/index.test.js

```diff
@@ -319,13 +319,51 @@ describe('GitRepository', () => {
       });
 
       it('stores the page ID', () => {
-        expect(commit.message).to.include(PAGE_ID);
+        expect(commit.body).to.include(PAGE_ID);
+      });
+    });
+
+    context('when there are one snapshot ID specified', () => {
+      const SNAPSHOT_ID = 'c01533c0e546ef430eea84d23c1b18a2b8420dfb';
+      const snapshotIds = [SNAPSHOT_ID];
+
+      before(async () => {
+        ({ id, isFirstRecord } = await subject.save(new Record({
+          serviceId: SERVICE_PROVIDER_ID,
+          documentType: DOCUMENT_TYPE,
+          pageId: PAGE_ID,
+          content: CONTENT,
+          fetchDate: FETCH_DATE,
+          mimeType: MIME_TYPE,
+          snapshotIds,
+        })));
+
+        ([commit] = await git.log());
+      });
+
+      after(async () => subject.removeAll());
+
+      it('stores snapshot ID', () => {
+        expect(commit.body).to.include(config.get('recorder.versions.storage.git.snapshotIdentiferTemplate').replace(SNAPSHOT_ID_MARKER, SNAPSHOT_ID));
```

Comment:
> This test is brittle as it reproduces the logic in the main code. It would be more stable if the expected result was hardcoded or loaded from fixtures.